lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -180,6 +181,12 @@ swapNodes = (targetBody, existingNodes, options) ->
         onNodeRemoved(existingNode)
   return
 
+removeNodes = (body, selector) ->
+  for node in findNodes(body, selector)
+    node.remove()
```

Comment:
`ChildNode.remove` doesn't work in IE. Use `node.parentNode.removeChild(node)`.

I would do this inline in `fetchHistory`:
```coffee
node.parentNode.removeChild(node) for node in findNodes(newDocument, '[data-ignore-from-transition-cache]')
```